### PR TITLE
Simplified Audiobook example

### PR DESCRIPTION
**Pull request by BigBlueHat**
The earlier JSON-LD version looked like it was extracted from the RDFa and had some cruft from that conversion left over. It's gone now, and should be simpler and more easily reused.

### CODE REVIEW

**Comment by BigBlueHat** [comment]
Well...I got inspired. 😁 The Audiobook examples now use the `url` to properly point at the actual Audiobook and `mainEntityOfPage` to point to the information page about the Audiobook.

I also chose a freely available audiobook from the kind people at Archive.org and LibriVox. It also happens to be Alice's Adventure's in Wonderland because Mad Hatters.

Cheers!
🎩

**Comment by RichardWallis** [comment]
A nice simple example.

I would tweak it slightly thus:
```
{
  "@context": "http://schema.org/",
  "type": "Audiobook",
  "@id": "https://archive.org/details/alices_adventures_1005_librivox",
  "abridged": "false",
  "author": "Lewis Carroll",
  "description": "Alice's Adventures in Wonderland (commonly shortened to Alice in Wonderland) is an 1865 novel written by English author Charles Lutwidge Dodgson under the pseudonym Lewis Carroll. It tells the story of a girl named Alice who falls down a rabbit hole into a fantasy world populated by peculiar and anthropomorphic creatures. The tale is filled with allusions to Dodgson's friends. The tale plays with logic in ways that have given the story lasting popularity with adults as well as children. It is considered to be one of the best examples of the "literary nonsense" genre, and its narrative course and structure have been enormously influential, especially in the fantasy genre. (Introduction by Wikipedia)",
  "datePublished": "2010-05-03",
  "duration": "PT2H31M21S",
  "encodingFormat": "video/mp4",
  "name": "Alice's Adventures in Wonderland",
  "playerType": "M4B compatible",
  "provider": "The Internet Archive",
  "publisher": "LibriVox",
  "readBy": "Eric Leach",
  "contentUrl": "https://archive.org/download/alices_adventures_1005_librivox/AlicesAdventuresInWonderlandV5_librivox.m4b",
  "mainEntityOfPage": "https://archive.org/details/alices_adventures_1005_librivox"
}
```
There is a difference between the Audiobook (CreativeWork) and the actual bytes of the media object.

File: data/ext/bib/bsdo-audiobook-examples.txt [hunk 1]
```diff
@@ -3,87 +3,70 @@ TYPES: Audiobook, readBy
...
 <div>
-<p>Author: Forsyth, Frederick, 1938-</p>
-<p>Title: The day of the jackal [downloadable audiobook] : a novel/ by Frederick Forsyth</p>
-<p>Ashland, OR : Blackstone Audio &#59; [Prince Frederick, Md.] : [Distributed by] OneClick Digital, 2011, p2009</p>
-<p>Connect to: <a href="http://overdrive.berkeleypubliclibrary.org/ContentDetails.htm?ID=E867D2C7-DE81-446B-8EC6-85057B587262">Downloadable audiobook</a></p>
-<p>Description: 1 sound file (13 hr., 31 min.) : digital</p>
-<p>Summary: The Jackal, the world's most cunning and revered assassin, is given a treacherous mission that could spell disaster for world diplomacy. Catching wind of a mysterious assassination plot, authorities throughout Europe mobilize a manhunt. However, without knowing the Jackal's true target, authorities are forced to wait until the clever killer makes his next move</p>
+<h1>Alice's Adventures in Wonderland</h1>
+<h2>by Lewis Carroll</h2>
+<p>provided by <a href="http://archive.org/">The Internet Archive</a>; publised by <a href="http://librivox.org/">LibriVox.org</a></p>
```
Comment (ostephens) [inline data/ext/bib/bsdo-audiobook-examples.txt]: Typo: publised

File: data/ext/bib/bsdo-audiobook-examples.txt [hunk 2]
```diff
@@ -3,87 +3,70 @@ TYPES: Audiobook, readBy
...
-  <span itemprop="provider">OneClick Digital</span>
-</p>
-<p>Connect to: <a itemprop="url" href="http://overdrive.berkeleypubliclibrary.org/ContentDetails.htm?ID=E867D2C7-DE81-446B-8EC6-85057B587262">Downloadable audiobook</a></p>
-<p>Description: <datetype itemprop="duration" content="P13H31M">1 sound file (13 hr., 31 min.) : digital</datetime></p>
-<p>Summary: <span itemprop="description">
-The Jackal, the world's most cunning and revered assassin, is given a treacherous mission that could spell disaster for world diplomacy. Catching wind of a mysterious assassination plot, authorities throughout Europe mobilize a manhunt. However, without knowing the Jackal's true target, authorities are forced to wait until the clever killer makes his next move</span></p>
+<h1 itemprop="title">Alice's Adventures in Wonderland</h1>
+<h2>by <span itemprop="author">Lewis Carroll</span></h2>
+<p>provided by <a href="http://archive.org/" itemprop="provider">The Internet Archive</a>;
+   publised by <a href="http://librivox.org/" itemprop="publisher">LibriVox.org</a></p>
```
Comment (ostephens) [inline data/ext/bib/bsdo-audiobook-examples.txt]: Typo: publised

File: data/ext/bib/bsdo-audiobook-examples.txt [hunk 3]
```diff
@@ -3,87 +3,70 @@ TYPES: Audiobook, readBy
...
-  <span property="provider">OneClick Digital</span>
-</p>
-<p>Connect to: <a property="url" href="http://overdrive.berkeleypubliclibrary.org/ContentDetails.htm?ID=E867D2C7-DE81-446B-8EC6-85057B587262">Downloadable audiobook</a></p>
-<p>Description: <datetime property="duration" content="P13H31M">1 sound file (13 hr., 31 min.) : digital</datetime></p>
-<p>Summary: <span property="description">
-The Jackal, the world's most cunning and revered assassin, is given a treacherous mission that could spell disaster for world diplomacy. Catching wind of a mysterious assassination plot, authorities throughout Europe mobilize a manhunt. However, without knowing the Jackal's true target, authorities are forced to wait until the clever killer makes his next move</span></p>
+<h1 property="title">Alice's Adventures in Wonderland</h1>
+<h2>by <span property="author">Lewis Carroll</span></h2>
+<p>provided by <a href="http://archive.org/" property="provider">The Internet Archive</a>;
+   publised by <a href="http://librivox.org/" property="publisher">LibriVox.org</a></p>
```
Comment (ostephens) [inline data/ext/bib/bsdo-audiobook-examples.txt]: Typo: publised

Comment (BigBlueHat) [reply data/ext/bib/bsdo-audiobook-examples.txt]: Clearly copy/paste works great on my computer... 😝

**Comment by BigBlueHat** [comment]
@RichardWallis so...this may be a bit [Range-14](https://en.wikipedia.org/wiki/HTTPRange-14)-like, but...

This trimmed example (from [your post earlier](https://github.com/schemaorg/schemaorg/pull/2232#issuecomment-488610056)--and on which I'm basing further commits to this PR) introduces some confusion (at least in my coffee-enhanced cranium):
```json
{
  "@context": "http://schema.org/",
  "type": "Audiobook",
  "@id": "https://archive.org/details/alices_adventures_1005_librivox",
  "contentUrl": "https://archive.org/download/alices_adventures_1005_librivox/AlicesAdventuresInWonderlandV5_librivox.m4b",
  "mainEntityOfPage": "https://archive.org/details/alices_adventures_1005_librivox"
}
```

That, afaict, says that `https://archive.org/details/alices_adventures_1005_librivox` is the Audiobook, but that the content of that URL is really at `https://archive.org/download/alices_adventures_1005_librivox/AlicesAdventuresInWonderlandV5_librivox.m4b` (which is one format/expression of the conceptual Audiobook) and that the "main entity" of `https://archive.org/details/alices_adventures_1005_librivox` is itself...

I realize that "naming things is hard," but I wanted to double check with you (et al) prior to making these changes.

This is more or less why I'd used `url` earlier and not `@id` or `contentUrl`, but again...I acknowledge this gets a bit Range-14 like...so input welcome! 😁

**Comment by BigBlueHat** [comment]
To avoid this being stalled indefinitely, @RichardWallis how do you feel about my removing the `@id` (leaving it unidentified...), but leave all the others in place?

**Comment by BigBlueHat** [comment]
@RichardWallis sorry for the wait on this one. I went with the `#work` suffix option. Please let me know if it needs anything else!

**Comment by danbri** [comment]
Github is reporting conflicts in data/ext/bib/bsdo-audiobook-examples.txt - could you take a look?

**Pull request merged**